lib/data_hygiene/organisation_reslugger.rb

```diff
@@ -21,7 +21,7 @@ def run!
       update_slug
       update_child_and_parent_organisations_in_search if organisation.is_a? Organisation
       update_users if organisation.is_a? Organisation
-      update_editions if organisation.is_a?(Organisation) || organisation.is_a?(WorldwideOrganisation)
+      update_editions if organisation.is_a?(Organisation)
```

Comment:
these lines all have the same condition now, what would you say to grouping them in a single `if`?

Reply:
Have grouped them all together now.